### PR TITLE
Questionnaire cypress test

### PR DESCRIPTION
**Pull request by mncharlton**
Adds a Cypress test for the entire questionnaire. This mocks the backend with a static JSON file. Using this means we don't need to have a backend running in CI (as we don't right now) but it also makes it important that we keep what's coming from the backend consistent. My next task will be adding an api test to the backend to make sure that we don't change what's coming from there without knowing that it will break the frontend.

CI will currently be failing on this because of an accessibility violation - raised this with design in Slack.

### CODE REVIEW

**Comment by mncharlton** [comment]
Noting that the JSON will soon be changing, once https://github.com/ClimateMind/climatemind-backend/pull/48 is merged we'll need to update this.

**Comment by NickCallaghan** [comment]
> Noting that the JSON will soon be changing, once [ClimateMind/climatemind-backend#48](https://github.com/ClimateMind/climatemind-backend/pull/48) is merged we'll need to update this.

I've a[ backend PR up at the minute](https://github.com/ClimateMind/climatemind-backend/pull/48) to change the shape of the Answers in the JSON. Better for the FE to get an array. We'll need to update the FE when this goes in which i suspect will break the current tests.

File: cypress/integration/questionnaire.spec.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ describe('Questionnaire loads and looks correct', () => {
 
   it('Can open the questionnaire', () => {
     cy.contains('Q1.').should('be.visible');
-    // cy.checkAccessibility(terminalLog) TODO: Failing
+    cy.checkAccessibility(terminalLog)
```
Comment (fe-mason) [inline cypress/integration/questionnaire.spec.js:18]: It looks like that this accessibility check failing is causing the rest of the test to not be run, so the percy snapshot isn't captured. Should we pull the accessibility checks out into another test or have the percy task complete first perhaps?

**Review by fe-mason — APPROVED** [review]
Added a rule to prevent contrast rules causing a11y failures for now, needs follow up as documented here https://www.pivotaltracker.com/story/show/174992068

Otherwise looks good to me!

**Pull request merged**